### PR TITLE
fix issue with create-element-to-jsx codemod

### PR DESCRIPTION
**Pull request by nickgervasi**

### CODE REVIEW

File: src/convert.js [hunk 1]
```diff
@@ -19,8 +19,8 @@ module.exports = function convert(cjsxPath) {
   const jsxPath = `${dirname(cjsxPath)}/${basename(cjsxPath, '.cjsx')}.jsx`;
   const jsSource = decaffeinate.convert(coffeeSource, { preferConst: true }).code;
   const jsxSource = prettier.format(
-    runTransform(createElementTransform, jsSource, jsxPath),
-    { singleQuote: true },
+    runTransform(createElementTransform, jsSource, jsxPath) || jsSource,
```
Comment (nickgervasi) [inline src/convert.js:22]: The create-element-to-jsx codemod returns null if no changes are made (see https://github.com/reactjs/react-codemod/issues/125).

**Comment by nickgervasi** [comment]
@rylanc I don't have write access to this repo for some reason so I couldn't make a pull-request without forking.

@nikhilmat I don't have admin access. If you get a chance, can you double check the permissions for this repo? Thanks!

**Comment by nikhilmat** [comment]
updated - added the `Developers` group with write access. @rylanc is the owner on this one and has admin rights, if you run into any other permissions issues on repos you've created you can just add that group or ping me and i can add it :)

**Review by rylanc — APPROVED** [review]

**Comment by zp-alfred** [comment]
Accepted by rylanc!

**Comment by nikhilmat** [comment]
also, 💚 you guys

**Pull request merged**